first.c

```diff
@@ -0,0 +1 @@
+coucou!
```

Comment:
I don't understand why you are saying coucou

Reply:
It's just for the fun